views/album.ejs

```diff
@@ -30,8 +32,8 @@
     <% photos.forEach(function(photo){ %>
       <li class="thumb">
         <a href="/<%= photo.url %>">
-          <img src="/<%= photo.src %>?tn=1" alt="<%= photo.name%>"><br />
-          <%= photo.name%>
+          <div class="image"><img src="/<%= photo.src %>?tn=1" alt="<%= photo.name%>"><br /></div>
+          <span class="caption"><%= photo.name%></span>
```

Comment:
Agree this needs greater semantics here - just not sure a span is the answer. 

Maybe a combination of `figure` and `figcaption`? 
http://html5doctor.com/the-figure-figcaption-elements/
Then, the <img> could have a parent container at a similar level to the above `div.image`